src/Compilers/Core/Portable/PEWriter/PeWriter.cs

```diff
@@ -8,11 +8,13 @@
 using System.Reflection.Metadata;
 using System.Reflection.Metadata.Ecma335;
 using System.Reflection.PortableExecutable;
+using System.Security.Cryptography;
 using System.Text;
 using System.Threading;
 using Microsoft.CodeAnalysis;
-using EmitContext = Microsoft.CodeAnalysis.Emit.EmitContext;
 using Microsoft.CodeAnalysis.Emit;
+using static Microsoft.Cci.SigningUtilities;
```

Comment:
Is `SigningUtilities` used here?

Reply:
Yes, it uses CalculateStrongNameSignatureSize.